test/powershell/Modules/Microsoft.PowerShell.Core/Job.Tests.ps1

```diff
@@ -1,49 +1,144 @@
 Describe "Job Cmdlet Tests" -Tag "CI" {
     Context "Simple Jobs" {
         AfterEach {
-            Get-Job | Remove-Job -force
+            Get-Job | Remove-Job -Force
         }
         BeforeEach {
-            $j = start-job -scriptblock { 1 + 1 }
+            $j = Start-Job -ScriptBlock { 1 + 1 } -Name "My Job"
         }
         It "Start-Job produces a job object" {
-            $j.gettype().fullname | should be "System.Management.Automation.PSRemotingJob"
+            $j | Should BeOfType "System.Management.Automation.Job"
+            $j.Name | Should Be "My Job"
         }
         It "Get-Job retrieves a job object" {
-            (Get-Job -id $j.id).gettype().fullname | should be "System.Management.Automation.PSRemotingJob"
+            (Get-Job -Id $j.Id) | Should BeOfType "System.Management.Automation.Job"
+        }
+        It "Get-Job retrieves an array of job objects" {
+            Start-Job -ScriptBlock { 2 * 2 }
+            $jobs = Get-Job
+            $jobs.Count | Should Be 2
+            foreach ($job in $jobs)
+            {
+                $job | Should BeOfType "System.Management.Automation.Job"
+            }
         }
         It "Remove-Job can remove a job" {
-            remove-job $j -force
+            Remove-Job $j -Force
             try {
-                get-job $j -ea Stop
+                Get-Job $j -ErrorAction Stop
                 throw "Execution OK"
             }
             catch {
-                $_.FullyQualifiedErrorId | should be "JobWithSpecifiedNameNotFound,Microsoft.PowerShell.Commands.GetJobCommand"
+                $_.FullyQualifiedErrorId | Should Be "JobWithSpecifiedNameNotFound,Microsoft.PowerShell.Commands.GetJobCommand"
             }
```

Comment:
I suggest using the excellent function [ShouldBeErrorId](https://github.com/PowerShell/PowerShell/blob/master/test/powershell/Common/Test.Helpers.psm1#L48)
(I have already prepared new PR #3161 to move existing tests to this pattern)
The new look of this test:
```powershell
            { Get-Job $j -ErrorAction Stop } | ShouldBeErrorId "JobWithSpecifiedNameNotFound,Microsoft.PowerShell.Commands.GetJobCommand" 
```
Only insert:
```powershell
Import-Module $PSScriptRoot\..\..\Common\Test.Helpers.psm1
```